### DISCUSSION TITLE
DOes this work whit a nRF52810 (Holyiot-2114)?

### DISCUSSION THREAD

**Discussion by barart** [capacitor-community/bluetooth-le#586] [Q&A] [answered]
I want to implement some module to my app, i want to see if this work with the Holyiot-2114 that has a nRF52810 chipset and a lis2dh12 accelerometer sensor

**Comment by pwespi** [comment, marked as answer]
This is a generic Bluetooth **Low Energy** plugins. So if you device supports Bluetooth **Low Energy**, which the nRF52810 does, then yes, it should work.

**Reply by barart** [reply]
The device has a button, do it read if the button is pressed?

**Reply by pwespi** [reply]
I don't know this device. The two ways to get data are `read` and `startNotifications`, but this depends on what's implemented in the device.
